### DISCUSSION TITLE
Shelly +1PM Mini supported?

### DISCUSSION THREAD

**Discussion by patpaat** [mongoose-os-apps/shelly-homekit#1555] [Q&A] [answered]
Hi there, is Shelly +1PM Mini also supported? 

I do not know, what kind of firmware I have to install or if it is possible. 

Can anyone help me? 

Thanks a lot!

**Comment by patpaat** [comment]
The OTA update doesn't work for me, so I need to know which one I need to download

**Comment by markirb** [comment]
It is not currently only gen3. See models on main page.

**Reply by patpaat** [reply]
do you think support for the +1PM Mini is still coming? or do I have to switch to the gen.3 devices? would be very annoying

**Reply by markirb** [reply, marked as answer]
I tried to do it quickly.

Can you try the build from:

https://github.com/mongoose-os-apps/shelly-homekit/tree/plus1pmmini

Access to file is via "checks"

**Reply by patpaat** [reply]
Thank you so much! 

I tried to install the firmware via [firmware ] on web (http://A.B.C.D/#/settings/firmware) and tried to import the homekit.zip but it does not update. i am on firmware 1.3.3. 

is it necessary to update to 1.4.4 first?

**Reply by patpaat** [reply]
i updated the firmware to 1.4.4 but it does not work for me. 

what do you mean „access to file via „checks”? maybe this is my problem?

**Reply by markirb** [reply]
Use this  (right click save as on Mac)
[shelly-homekit-ShellyPlus1PMMini.zip](https://github.com/user-attachments/files/17739261/shelly-homekit-ShellyPlus1PMMini.zip)

**Reply by patpaat** [reply]
you are simply the best – it‘s all done! 
keep up the good work! many, many thanks!
